lib/directory.coffee

```diff
@@ -177,6 +177,9 @@ class Directory
       stat = fs.lstatSyncNoException(fullPath)
       symlink = stat.isSymbolicLink?()
       stat = fs.statSyncNoException(fullPath) if symlink
+      statFlat = _.pick stat, _.keys(stat)...
+      for i in ["atime", "birthtime", "ctime", "mtime"]
```

Comment:
Minor this, but this is not an index, so I think the `i` should be changed to `key`, `property`, `type`, or some other colloquialism that is more appropriate than `i`.